cves/CVE-2017-16840.yml

```diff
@@ -127,7 +133,10 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    Since the earliest vcc, the time till the fix was less than a month, about 11 days. 
```

Comment:
until or til, not till

Reply:
Please choose until. This is a relatively professional document.